gui/wxpython/history_catalog/catalog.py

```diff
@@ -0,0 +1,116 @@
+"""
+@package history_catalog::catalog
```

Comment:
I am not sure I like catalog, maybe browser?I think I would prefer just `history` for the folder.